### PR TITLE
LESQ-706: Fix the bread crumb links on the share and download pages

### PR DESCRIPTION
**Pull request by k-huggs**
## Description

Music year: 2021 - https://www.youtube.com/watch?v=tQ0yjYUFKAE

- Fixes the the bread crumb links on the headers of both Downloads and Share pages

## Issue(s)

Fixes #LESQ-706

## How to test

1. Go to https://deploy-preview-2322--oak-web-application.netlify.thenational.academy
2. Click on \_\_\_\_\_\_
3. You should see \_\_\_\_\_\_

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist

- [ ] Added or updated tests where appropriate
- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner
- [ ] Does this PR update a package with a breaking change

### CODE REVIEW

File: src/components/TeacherViews/LessonShare/LessonShare.view.tsx [hunk 1]
```diff
@@ -47,12 +51,18 @@ type LessonShareProps =
   | {
       isCanonical: false;
       lesson: LessonPathway & {
+        isSpecialist: false;
+        developmentStageTitle?: string | null;
```
Comment (njblore) [inline src/components/TeacherViews/LessonShare/LessonShare.view.tsx:55]: this will never be present for non specialist right?

**Review by njblore — APPROVED** [review]

**Review by jack-skerrett-oak — APPROVED** [review]
Breadcrumbs still work in legacy/new browsing journeys, canonical lessons and webinar/blogs.

File: src/components/TeacherViews/LessonShare/LessonShare.view.tsx (hunk 1 above)
Comment (k-huggs) [reply src/components/TeacherViews/LessonShare/LessonShare.view.tsx:55]: No never present, but the TS checker wouldn't let the isSpecialist check work kept throwing up a type error

**Pull request merged**